### PR TITLE
Added copy function with shiftclick

### PR DESCRIPTION
**Pull request by blackmagic919**

### CODE REVIEW

**Comment by cirex-web** [comment]
You can copy to clipboard by pressing ctrl/cmd+C after selecting some rows in the table

File: src/features/WordTable/TableHeader/index.tsx [hunk 1]
```diff
@@ -49,7 +60,9 @@ export const TableHeader = ({
             </Button>
 
             <Button
-              onMouseDown={() => saveFlashcards(folderName, cards)}
+              onMouseDown={(event) => event.shiftKey || event.metaKey 
+                ? copyFlashcards(folderName, cards.reverse())
```
Comment (cirex-web) [inline src/features/WordTable/TableHeader/index.tsx]: This feels like a hacky fix... `filteredCards` already has the cards reversed. Maybe that can be passed as a prop instead.

**Review by cirex-web — CHANGES_REQUESTED** [review]

File: src/features/WordTable/TableHeader/index.tsx (hunk 1 above)
Comment (blackmagic919) [reply src/features/WordTable/TableHeader/index.tsx]: Lol I put like 0 thought into this problem yeah that's valid

**Review by cirex-web — APPROVED** [review]
@blackmagic919 I just cleaned up some of your code - Let me know if you have any questions!

**Pull request merged**